### PR TITLE
Feature/data pipeline

### PR DESCRIPTION
**Pull request by rmania**

### CODE REVIEW

File: guides/setting-up.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+explains: A working guide for analists in the DataLab 
+---
+
+# Setting up
```
Comment (bvhme) [inline guides/setting-up.md:5]: I suggest using a more descriptive title such as 'Setting up an data analysis project', to give a bit more context

File: guides/setting-up.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Ideally this file starts by ignoring everything (`*`) and then explicitly add the files that belong to the build
+by means of exceptions (e.g. `!**\*.py` to include all python source files).
+Most often you will however find .dockerignore files that only specify files that are not part of the build. 
+
+## Where do we store code
+
+Every project should share reproducible code. We distinguish between sensitive projects (like for privacy sensitive projects) and 
+projects with data that is not particularly sensitive.
+
+- If your project contains contains sensitive code push your code to https://git.datapunt.amsterdam.nl/
```
Comment (bvhme) [inline guides/setting-up.md:33]: Note that this is internal and only approachable from the inside.

File: guides/setting-up.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- otherwise to the relevant code repository on https://github.com/amsterdam
```
Comment (bvhme) [inline guides/setting-up.md:34]: I would reorganise this as 

> everything is on github.com/amsterdam with the exception of privacy-sensitive projects that are hosted on our own closed git at git.datapunt.amsterdam.nl

File: guides/setting-up.md (hunk 1 above)
Comment (rmania) [reply guides/setting-up.md:5]: Nice will change this.

**Comment by Lytrix** [comment]
@bvhme Thanks! We've created a clean feature/guide-data-pipeline branch. We will copy the changes there next week.

**Pull request closed without merging**